.gitignore

```diff
@@ -1,3 +1,5 @@
 /node_modules/
+dist/
```

Comment:
Is this BC?  Users expect `codejar.js` at the root.

Reply:
BC? 
All bundled files go to dist folder, but whit package.json fields users doesn't see anything different 

`"main": "dist/codejar.js",
  "module": "dist/codejar.module.js",
  "unpkg": "dist/codejar.umd.js",`

Reply:
BC == Breaking Change.

What is users import files directly? For example via `import {} from 'https://unpkg.com/@medv/codejar@1.0.8/codejar.js'`?